src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Security/RequestHandlerCheckAccessTests.cs

```diff
@@ -50,7 +50,7 @@ void Dfs(Assembly a)
             Dfs(typeof(SecurityProvider).Assembly);
 
             return assemblies.SelectMany(a => a.GetTypes())
-                .Where(t => t.IsClass && !t.IsAbstract && t.GetInterfaces().Any(i => i.IsGenericType && i.GetGenericTypeDefinition() == typeof(IRequestHandler<,>)))
+                .Where(t => t.IsClass && string.Compare("LatestSchemaVersionHandler", t.Name, StringComparison.OrdinalIgnoreCase) != 0 && !t.IsAbstract && t.GetInterfaces().Any(i => i.IsGenericType && i.GetGenericTypeDefinition() == typeof(IRequestHandler<,>)))
```

Comment:
Shouldn't be needed right?

Reply:
There is also an `_exceptions` list above, would be a better place for this

Reply:
Its not needed as the handler is now moved to sql package